translations/base-en.yaml

```diff
@@ -1209,38 +1209,37 @@ tips:
     - You don't need to divide up items evenly for full efficiency.
     - Holding <b>SHIFT</b> will activate the belt planner, letting you place long lines of belts easily.
     - Cutters always cut vertically, regardless of their orientation.
-    - To get white mix all three colors.
+    - To get white, mix all three colors.
     - The storage buffer prioritises the left output.
     - Invest time to build repeatable designs - it's worth it!
-    - Holding <b>CTRL</b> allows to place multiple buildings.
+    - Holding <b>SHIFT</b> lets you place multiple buildings in one click.
     - You can hold <b>ALT</b> to invert the direction of placed belts.
     - Efficiency is key!
     - Shape patches that are further away from the hub are more complex.
     - Machines have a limited speed, divide them up for maximum efficiency.
     - Use balancers to maximize your efficiency.
     - Organization is important. Try not to cross conveyors too much.
-    - Plan in advance, or it will be a huge chaos!
+    - Plan in advance, or it will be a mess!
     - Don't remove your old factories! You'll need them to unlock upgrades.
-    - Try beating level 20 on your own before seeking for help!
+    - Try beating level 20 or 26 on your own before seeking for help!
     - Don't complicate things, try to stay simple and you'll go far.
-    - You may need to re-use factories later in the game. Plan your factories to be re-usable.
+    - You may need to reuse factories later in the game. Build your factories to be reusable.
     - Sometimes, you can find a needed shape in the map without creating it with stackers.
     - Full windmills/pinwheels can never spawn naturally.
     - Color your shapes before cutting for maximum efficiency.
     - With modules, space is merely a perception; a concern for mortal men.
     - Make a separate blueprint factory. They're important for modules.
     - Have a closer look at the color mixer, and your questions will be answered.
     - Use <b>CTRL</b> + Click to select an area.
-    - Building too close to the hub can get in the way of later projects.
+    - Building too close to the hub can get in the way for later projects.
```

Comment:
I think most would use "get in the way of"; "for" makes it sort of sound like building too close and later projects work together